popout.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en-US" ><head>
+	<meta charset="UTF-8"/>
+	<link rel="icon" type="image/png" href="./favicon.ico"/>
+	<link rel="stylesheet" type="text/css" href="css/main.css"/>
+	<link rel="stylesheet" type="text/css" href="css/popout.css"/>
+	<script src="./js/module-shim.mjs" type="module"></script>
+	<script src="https://valarnin.github.io/oblivion-save-reader/dist/bundle.savefile.js" defer></script>
+    <script src="https://valarnin.github.io/oblivion-save-reader/dist/bundle.record.js" defer></script>
```

Comment:
only need these if we're going to be parsing saves on this page

Reply:
removed them, thanks for point that out